### ISSUE TITLE
[Bug]: ZZZ Cache Updates is checking for broken files

### ISSUE THREAD

**Issue by Caelum008** [CollapseLauncher/Collapse#616]
### Affected Version

1.82.3-Pre

### Bug Behavior

Not sure if this was already reported but instead of checking for cache updates, I think it's checking for broken files like how the game repair functions.

### Expected Behavior

Should check for cache updates instead of broken files.

### Steps to reproduce

1. Reinstalled

### Screenshot(s)

![image](https://github.com/user-attachments/assets/652a4e19-3abb-484a-8c35-99f440281730)

![image](https://github.com/user-attachments/assets/f8ac56b3-43b3-462b-aa57-b2ea2cae6ef4)

**Comment by bagusnl** [comment]
Since ZZZ Cache checking function uses inherited functions from the Repair counterpart and I think we forgor to put different text on cache, but the function should be different in the backend

TL;DR its a UI/UX issue rather than wrong check
We'll try to fix the text assignment

**Comment by Caelum008** [comment]
Ohh. I see. Thank you~

**Comment by Cryotechnic** [comment]
Issue has been resolved in latest preview and stable releases. Thanks for your report!
